crypto/cms/cms_ess.c

```diff
@@ -19,6 +19,15 @@
 #include "crypto/ess.h"
 #include "crypto/cms.h"
 
+/*
+ * Returns < 0 if attribute is not found, 1 if found, or 
+ * -1 on attribute parsing failure.
+ */
+int cms_signerinfo_get_signing_cert_v2(CMS_SignerInfo *si,
```

Comment:
If this function is only used in this file then they should be static.
If they are used by other files then they should be in a header.

Reply:
:+1: